<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupHost</class>
 <widget class="QWidget" name="SetupHost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>375</width>
     <height>195</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>User name</string>
      </property>
      <property name="buddy">
       <cstring>userEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Password</string>
      </property>
      <property name="buddy">
       <cstring>passwordEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="remPass">
      <property name="text">
       <string>Remember Password</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Host</string>
      </property>
      <property name="buddy">
       <cstring>hostEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="userEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="passwordEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="hostEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="useSSL">
      <property name="text">
       <string>Use SSL</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="nonStandard">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>use non-standard port</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="portEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="done">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>saveData()</slot>
 </slots>
</ui>
